### PR TITLE
Go 1.14

### PR DESCRIPTION
**Pull request by dgageot**
Switch to Go 1.14

### CODE REVIEW

**Comment by nkubala** [comment]
@dgageot can you rebase?

File: examples/custom/build.sh [hunk 1]
```diff
@@ -2,7 +2,9 @@
 set -e
 
 if ! [ -x "$(command -v ko)" ]; then
-    GO111MODULE=on go get -mod=readonly github.com/google/ko/cmd/ko
```
Comment (tejal29) [inline examples/custom/build.sh]: making change in examples. is this intentional?
Shd you make this in integration?

**Comment by dgageot** [comment]
@nkubala Can you take another look, please?

**Comment by mjgallag** [comment]
@nkubala what are the chances of this being released as a patch to v1.5.0? See https://github.com/Homebrew/homebrew-core/pull/51233#issuecomment-597963608.

**Comment by nkubala** [comment]
@mjgallag looks like this was addressed by the homebrew folks in that PR. we'll ship this out with the release this week.

**Pull request merged**